docs/universal_transfer_operator/operator.rst

```diff
@@ -83,3 +83,38 @@ Comparison with traditional transfer Operator
            :language: python
            :start-after: [START howto_transfer_data_from_s3_to_snowflake_using_S3ToSnowflakeOperator]
            :end-before: [END howto_transfer_data_from_s3_to_snowflake_using_S3ToSnowflakeOperator]
+
+
+.. _load_strategies:
+
+Load Strategies
+~~~~~~~~~~~~~~~
+
+Following are the load strategies supported for :ref:`table` transfers:
```

Comment:
```suggestion
Following are the load strategies supported when :ref:`table` is the destination dataset:
```

Reply:
Also, we should change if_exists to some name that takes into account that it's talking about destination and table/file if they have distinct strategies in future.